themes/openy_themes/openy_rose/scss/modules/_paragraphs.scss

```diff
@@ -490,6 +490,26 @@
         object-fit: cover;
       }
     }
+
+    @include breakpoint($screen-desktop) {
+      &.video {
```

Comment:
Is it possible to add this style to all OpenY themes?
Would be cool to keep sass and generated css in module and include it globally as it is already done for some other modules https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/scss/modules.scss
And included by every theme automatically for compiling